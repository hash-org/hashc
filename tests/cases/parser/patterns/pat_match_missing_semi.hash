// run=fail, stage=parse

other_func := (t: str) => {
    match t {
        "one" => 1;
        "two" => 2;
        "three" => 3;
        "four" | "five" if var => 4.5;
        alpha if var == 2 => {
            print(alpha)
        };
        _ => -1
    };

    return alpha;
};
