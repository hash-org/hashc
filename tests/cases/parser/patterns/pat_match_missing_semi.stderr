error: unexpectedly reached the end of input
  --> $DIR/pat_match_missing_semi.hash:12:16
11 |           };
12 |           _ => -1
   |                  ^ 
13 |       };
   = help: consider adding a `;`
